projects/packages/my-jetpack/_inc/components/product-card/index.jsx

```diff
@@ -176,7 +229,20 @@ const ProductCard = props => {
 					<Text variant="title-medium">{ name }</Text>
 					{ menuIsActive && icon }
 				</div>
-				{ menuIsActive ? <Menu items={ menuItems } onManage={ onManage } /> : icon }
+				{ menuIsActive ? (
+					<Menu
+						status={ status }
```

Comment:
We need to use this one in `Menu` to remove the `Manage` button if the status isn't `active`. Since the user could manage nothing if it isn't active.